app/backend/main.py

```
@@ -56,6 +56,50 @@ async def send_wrapper(message: dict) -> None:
+    storage_path = Path(settings.STORAGE_PATH).resolve()
+    if not storage_path.is_dir():
+        return
+    if _async_session is None:
+        logger.debug("[Orphan] DB 세션 없음 — 건너뜀")
+        return
+
+    try:
+        async with _async_session() as session:
+            result = await session.execute(select(Project.game_id))
+            db_game_ids: set[str] = set(result.scalars().all())
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)